### PR TITLE
Murlocs page expansion into all associated acts

### PR DESCRIPTION
**Pull request by WBTGS95**

### CODE REVIEW

**Review by alxndr — COMMENTED** [review]
gonna rename the folder so the URL becomes "/associated-music" how's that sound?

File: the-murlocs/index.md [hunk 1]
```diff
@@ -1,8 +1,50 @@
...
+
+### Bullant  
+Bullant is the solo techno/IDM production project of Joey. His debut album _Tyson, Crying_ shows the learning process of Ableton in real time, unable to save a project using the free version of the software, each song was completed in a single session under the stairs at the studio. Beginning in 2019, one could infer that this project was the offshoot of the electronic experiments of King Gizzard such as can be heard at the end of [_Fishing For Fishies_](https://kglw.net/releases/fishing-for-fishies), or in [_Made In Timeland_](https://kglw.net/releases/made-in-timeland).
+
+### CAVS  
+In 2021, Cavs released his debut solo self-titled drumming album. A pure expression of his drumming ability, CAVS shows off chops in various percussive styles outside of what he usually gets to play with King Gizzard.
```
Comment (alxndr) [inline the-murlocs/index.md:21]: Wanna link to the Discogs page for these albums? e.g. https://www.discogs.com/master/2280727-Cavs-Cavs

Also should the album names all be italicized?

Comment (WBTGS95) [reply the-murlocs/index.md]: "/associated-music" Sounds good. Discogs page and italicised titles also good, I'll get to those as a finish the body.

**Comment by alxndr** [comment]
this will address #14

**Pull request merged**